modules/Bio/EnsEMBL/Compara/Utils/FlatFile.pm

```diff
@@ -85,6 +86,7 @@ sub map_row_to_header {
     return $row;
 }
 
+
```

Comment:
Does this line need to be added here? Could it be left out?

Reply:
Yes, thanks!